### PR TITLE
Add NOX Unified Target

### PR DESCRIPTION
**Pull request by Asizon**
Adding Airbot NOX Unified Target configuration

### CODE REVIEW

File: configs/default/AIRB-NOX.config [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+dma pin B08 0
+# pin B08: DMA1 Stream 7 Channel 2
+dma pin B01 0
+# pin B01: DMA1 Stream 2 Channel 5
+dma pin A00 0
+# pin A00: DMA1 Stream 5 Channel 3
+
+# feature
+feature -RX_PARALLEL_PWM
+feature RX_SERIAL
```
Comment (IvoFPV) [inline configs/default/AIRB-NOX.config]: Should have `feature OSD` as well.

Comment (etracer65) [reply configs/default/AIRB-NOX.config]: To be consistent with the legacy target it should also have:
```
feature SOFTSERIAL
feature ESC_SENSOR
```

**Comment by Asizon** [comment]
Thanks guys, done!

**Comment by Asizon** [comment]
@etracer65 ready for merged

**Pull request merged**